easybuild/easyconfigs/a/ASE/ASE-3.17.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,60 @@
+easyblock = 'PythonBundle'
+
+name = 'ASE'
+version = '3.17.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('Tkinter', '%(pyver)s'),
+    ('matplotlib', '3.0.3', '-Python-%(pyver)s'),
```

Comment:
You need ('SciPy-bundle', '2019.03') in the dep list. That's where numpy/scipy is.

Reply:
No, matplotlib depends on SciPy-bundle.  Is that not enough?

Reply:
ASE has a direct dependency on numpy/scipy and thus SciPy-bundle should be explicitly listed.

Reply:
OK, I'll fix it then.